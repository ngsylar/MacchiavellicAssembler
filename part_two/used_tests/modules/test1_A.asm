; crknjkrcn
dffs gjsvjg           jnkj
ndkjdn: gdkhbs
jndkjdn:
MOD_A: BEGIN
SECTION DATA
	N: SPACE
	ONE: CONST 1


	; kdjnsfkj ssds


SECTION TEXT
	MOD_B: EXTERN
	PUBLIC FAT
	PUBLIC N
	INPUT N+1
	LOAD N + 3
	FAT: SUB ONE
	JMPZ FIM
	JMP MOD_B
	FIM: OUTPUT N

	STOP
	COPY n+1+14,n+2
END

kmdllkd
; rscsrc