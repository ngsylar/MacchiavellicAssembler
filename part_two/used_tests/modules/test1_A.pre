JNDKJDN: MOD_A: BEGIN
SECTION TEXT
MOD_B: EXTERN
PUBLIC FAT
PUBLIC N
INPUT N+1
LOAD N + 3
FAT: SUB ONE
JMPZ FIM
JMP MOD_B
FIM: OUTPUT N
STOP
COPY N+1+14,N+2
SECTION DATA
N: SPACE
ONE: CONST 1
END