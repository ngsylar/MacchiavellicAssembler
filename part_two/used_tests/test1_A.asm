; crknjkrcn
dffs gjsvjg           jnkj
ndkjdn: gdkhbs
jndkjdn:
MOD_A: BEGIN
SECTION DATA
	N: SPACE
	ONE: CONST 1


	; kdjnsfkj ssds


SECTION TEXT
	MOD_B: EXTERN
	PUBLIC FAT
	PUBLIC N
	INPUT N
	LOAD N
	FAT: SUB ONE
	JMPZ FIM
	JMP MOD_B
	FIM: OUTPUT N

	STOP
END

kmdllkd
; rscsrc