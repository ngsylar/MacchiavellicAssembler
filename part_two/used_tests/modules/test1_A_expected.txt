12 17
10 16
2 17
8 10
5 0
13 16
14
9 17 18
0
1